easybuild/easyconfigs/p/PGI/PGI-16.1-GCC-4.9.2-2.25.eb

```diff
@@ -0,0 +1,26 @@
+name = 'PGI'
+version = '16.1'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgicdk-%(version_major)s%(version_minor)s.tar.gz']
+checksums = ['442c044b9690a84b6600cd6919480bcb']
+
+gccver = '4.9.2'
+binutilsver = '2.25'
+versionsuffix = '-GCC-%s-%s' % (gccver, binutilsver)
+
+dependencies = [
+    ('GCCcore', gccver),
+    ('binutils', binutilsver, '', ('GCCcore', gccver)),
+    ('numactl', '2.0.11', '', ('GCCcore', gccver)),
+]
+
+# license file
+import os
+license_file = os.path.join(HOME, "licenses", "pgi", "license.dat")
```

Comment:
@valtandor avoiding `import` statements of 'portability' wins here, since the `/` directory separator argument only comes up with Windows vs Unix-based OSs (in easyblocks/framework it makes sense to stick to `os.path.join` though)